### PR TITLE
Build operator docs

### PR DESCRIPTION
**Pull request by JohannMoller**
Closes ~#ISSUE_NUMBER~

### What are the changes and their implications?
This PR adds documentation for the `_build` operator.

## Checklist

- [x] Pull request is made to the "develop" branch
- [ ] Tests added
- [x] Documentation added/updated
- [x] Code has been formatted with Prettier
- [x] Edits from maintainers are allowed

### CODE REVIEW

File: pnpm-lock.yaml [hunk 1]
```diff
@@ -1292,6 +1292,8 @@ importers:
       '@lowdefy/build': 4.0.0-rc.0
       '@lowdefy/client': 4.0.0-rc.0
       '@lowdefy/connection-axios-http': 4.0.0-rc.0
+      '@lowdefy/connection-mongodb': 4.0.0-rc.0
```
Comment (SamTolmay) [inline pnpm-lock.yaml:1295]: Please revert this

File: packages/docs/operators/_build.yaml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+          ```
+        description: |
+          The `_build.env` method reads from your environment variables. The return value of `_build.env` will form part of the build output, so the [`_secret`](/_secret) operator should rather be used for reading any secrets.
+        examples: |
+          ###### Read environment from environment variables:
+          With the following development and production environment variables, we can use the `_build.env` method to determine which environment we are in.
+
+          `Development environment .env`
+          ```
+          MY_ENVIRONMENT="Development"
```
Comment (SamTolmay) [inline packages/docs/operators/_build.yaml:39]: Environment vars are normally lowercase, so I think this should rather be `production` and `development`

File: pnpm-lock.yaml (hunk 1 above)
Comment (JohannMoller) [reply pnpm-lock.yaml:1295]: All the changes to the file?

Comment (JohannMoller) [reply pnpm-lock.yaml]: Changes made in [6dbcdb5](https://github.com/lowdefy/lowdefy/pull/1469/commits/6dbcdb58656dc5e6d004c7c6f14db5b52da6ddaf)

File: packages/docs/operators/_build.yaml (hunk 2 above)
Comment (JohannMoller) [reply packages/docs/operators/_build.yaml]: Changes made in [bafc31d](https://github.com/lowdefy/lowdefy/pull/1469/commits/bafc31d6dde5558f5ef0025955a7cb9b326f3f06)

**Review by Gervwyk — APPROVED** [review]

**Pull request merged**